templates/repo/issue/card.tmpl

```diff
@@ -0,0 +1,66 @@
+{{with .Issue}}
+	{{if eq $.Page.Project.CardType 1}}{{/* Images and Text*/}}
+		<div class="card-attachment-images">
+			{{range (index $.Page.issuesAttachmentMap .ID)}}
+				<img src="{{.DownloadURL}}" alt="{{.Name}}" />
+			{{end}}
+		</div>
+	{{end}}
+	<div class="content gt-p-0">
+		<div class="gt-df gt-items-start">
+			<div class="issue-card-icon">
+				{{template "shared/issueicon" .}}
+			</div>
+			<a class="issue-card-title muted issue-title" href="{{.Link}}">{{.Title | RenderEmoji ctx | RenderCodeBlock}}</a>
+			{{if and $.isPinned $.Page.isRepoAdmin}}
```

Comment:
If I understand this code correctly, there's a check missing.
Otherwise, you can unpin an issue even from a project if I see that correctly.
I don't think that's intended.

Reply:
isPinned isn't an issue parameter, it is being passed manually then calling the template. Maybe I can rename it to isPinnedIssueCard or something like that to avoid confusion....

Reply:
Done